test/task.js

```diff
@@ -64,14 +71,18 @@ describe('task', function() {
   });
 
   it('should get a task that was registered', function(done) {
-    taker.task('test1', noop);
+    expect(taker.task('test1', noop)).to.be.undefined();
```

Comment:
Because I wanted to check that `task` returned nothing when working as a setter.
Is it better to change back to original?